### ISSUE TITLE
Please add some example images

### ISSUE THREAD

**Issue by peterdk** [nanihadesuka/LazyColumnScrollbar#36]
It would be very useful to have some example images in the readme, of your library implemented. Now I have no clue if this library indeed does what I need it to.

**Comment by nanihadesuka** [comment]
Is a scrollbar, what do you expect it to look like? 🤔

**Comment by nanihadesuka** [comment]
I don't think this library in specific really needs images. The README is clear enough. Closing.

**Comment by peterdk** [comment]
Sorry for the late reply. 

I had found it useful to see some customization options visually. Like scrollbar size, maybe the animation, etc. It's a UI library after all. But ok.

**Comment by jsericksk** [comment]
I completely agree with @peterdk. I just opened the repository and it's very strange not to see images to visually exemplify. I believe this issue shouldn't have been closed without being resolved. What does it look like with different settings? What is its animation like? These questions should be answered with images. It's a UI library, we want to see the UI.

I think it would be very important to add images.
